master/buildbot/test/fake/endpoint.py

```diff
@@ -38,7 +39,7 @@ class TestsEndpoint(base.Endpoint):
 
     def get(self, resultSpec, kwargs):
         # results are sorted by ID for test stability
-        return defer.succeed(sorted(testData.values(), key=lambda v: v['id']))
+        return defer.succeed(sorted(list(itervalues(testData)), key=lambda v: v['id']))
```

Comment:
Yes. I wasn't sure if ```sorted``` worked on dict_items objects. It does, so I'll remove the list